garrysmod/lua/vgui/dcolorbutton.lua

```diff
@@ -27,11 +27,15 @@ function PANEL:IsDown()
 
 end
 
-function PANEL:SetColor( color )
+function PANEL:SetColor( color, displayTooltip )
 
-	local colorStr = "R: " .. color.r .. "\nG: " .. color.g .. "\nB: " .. color.b .. "\nA: " .. color.a
-
-	self:SetTooltip( colorStr )
+	if ( displayTooltip == true ) then

```

Comment:
To support the old behaviour, you should check 
```lua
if displayTooltip ~= false then
```
or use 
```lua
if displayTooltip == nil then 
    displayTooltip = true 
end
```

Reply:
I changed it so everything works the same as long as you don't add a false after the color.